manifests/plugin_guests.pp

```diff
@@ -13,6 +13,7 @@
   $guest_merch_enums = undef,
 
   $auction_start = '',
+  $rock_island_groups = undef,
```

Comment:
I think leaving this undef will break for any of our deployments that don't have `uber::plugin_guests::rock_island_groups` defined in their `yaml` files. My preference would be to make this default to `'bands,'`.